### DISCUSSION TITLE
2.1.1 release

### DISCUSSION THREAD

**Discussion by eschutho** [apache/superset#23993] [General]
This is a discussion for any recommendations for bug fixes to be released with 2.1.1. Please post any PRs below that are bug fixes, not features. If you have a bug issue, you can paste it below, but we'll need community help to address it by the release. 

Currently tagged to go into 2.1.1: https://github.com/apache/superset/pulls?q=is%3Apr+label%3A2.1.1

**Comment by kartikdhar** [comment]
Please look into this: https://github.com/apache/superset/issues/23645

unix timestamp datetime objects are not working with the new bar and line / area chart components apparently

**Comment by quasio007** [comment]
Can we also include [#23582](https://github.com/apache/superset/pull/23582)

This is error with imported dashboards (Ensure old datasource ids are not referenced in imported charts)

**Reply by eschutho** [reply]
Thanks, I tagged it for 2.1.1.

**Comment by santosh-sahoo334** [comment]
Please look into this - https://github.com/apache/superset/discussions/23928
This is a bug in Deck.gl scatter plot charts. Thanks!

**Reply by eschutho** [reply]
If we get a PR for this, we can pull it into the next release.

**Reply by santosh-sahoo334** [reply]
Hi @eschutho Thanks for the follow up. I dont have a solution right now. @rusackas requested @kgabryje to look at it. Here is the related issue & discussion.
https://github.com/apache/superset/issues/24016

**Comment by iercan** [comment]
This bug is blocker for us. Please include #23648

**Reply by eschutho** [reply]
Looks like @rusackas mentioned that this flag will be deprecated in 3.0. I don't see that there's a PR for this yet, but if someone submits one, we can pull it into the next patch release.

**Comment by paulinjo** [comment]
Please consider adding #20604

**Reply by eschutho** [reply]
We don't have a PR ready for this issue, but plan to release something in 3.0.

**Comment by D3nn3** [comment]
Could we look into this one? #23092 
Don't know if I was the only one that noticed a performance regression with 2.1.0 but this was mainly the reason why we still run 2.0.1 at my company.

**Comment by saghatelian** [comment]
It would be great if this bug could be fixed.
https://github.com/apache/superset/issues/23167

**Reply by eschutho** [reply]
Looks like there's not a PR yet, but hopefully someone can submit a PR for the next release

**Comment by saghatelian** [comment]
Does this fix have a chance of making it into the release?
https://github.com/apache/superset/pull/23318

**Reply by eschutho** [reply]
Thanks, I added this to the list.

**Reply by eschutho** [reply]
Actually, when cherry-picking this PR, I noticed that has changes across 55 files, so it will be too large to cherry into the release as a bug fix, but it will be in 3.0 shortly.

**Comment by eschutho** [comment]
We're starting to package up the 2.1.1 release, so for now, this looks like all the PRs we're going to be able to add to this patch release. Thank you everyone for all the suggestions!

**Comment by santoshmallah** [comment]
Please look into this - https://github.com/apache/superset/issues/24199
embedded dashboard in iframe working in 2.0.1 
but now I am getting dashboard load issue after upgrading from 2.0.1=>2.1.0

Thanks!

**Comment by mdeshmu** [comment]
Please look into https://github.com/apache/superset/issues/24228

**Comment by dlabmx** [comment]
Please include this #23554

**Reply by eschutho** [reply]
We're waiting for a test for this PR in order to merge it.

**Comment by xaviRodri** [comment]
Would be amazing to add #23565
Adding filters is a must for us! 🙏

**Comment by mdeshmu** [comment]
Please include https://github.com/apache/superset/pull/23319

**Reply by eschutho** [reply]
Hi @mdeshmu, thanks for the suggestion. Feature work will go into the next major/minor release rather than this one, which is only for bug fixes.

**Reply by mdeshmu** [reply]
Thanks @eschutho for reply

**Comment by mdeshmu** [comment]
I request to include https://github.com/apache/superset/pull/23400 and https://github.com/apache/superset/pull/23566 in this release as the bug is causing a very unpleasant experience for our end users.

**Reply by eschutho** [reply]
Thanks for flagging those PRs. I've already bundled the release, but if there is a chance to add more PRs, I'll pull these in.

**Reply by mdeshmu** [reply]
please add #23801 as well, please

**Reply by mdeshmu** [reply]
@eschutho did these PRs make it to the 2.1.1rc1?

**Reply by mdeshmu** [reply]
@eschutho @michael-s-molina Based on the discussion in the 2.1.1rc1 voting email thread, looks like we have a possibility of 2.1.1rc2. Can you please include the above PRs in 2.1.1rc2? Thanks.

**Reply by eschutho** [reply]
@mdeshmu yes, I'll check on those PRs for rc2.

**Reply by eschutho** [reply]
@mdeshmu I've added all three of those PRs into rc2

**Reply by mdeshmu** [reply]
Thank you @eschutho. Really appreciate your efforts for the release.

**Comment by sqja** [comment]
Hello,
Would it be possible to include a fix for this regression #21508 which was spotted in 2.1.0 please? 
Due to it we cannot upgrade to version > 2.0.1 at this moment

**Reply by D3nn3** [reply]
Might be related (or even a duplicate): #23197 
And I agree, this is one reason why we didn't update to 2.1.0 so far

**Reply by eschutho** [reply]
I haven't seen a PR for this issue yet so I'm going to skip this for now, and we'll see if we can get one into a future patch release.

**Comment by hot123s** [comment]
Would it be possible to add https://github.com/apache/superset/pull/24005 and https://github.com/apache/superset/pull/24287 to 2.1.1 release.

**Reply by eschutho** [reply]
This looks like a great feature, but we can only pull in patch/bug fixes into this 2.1.1 release. We'll get this PR into 3.0 which is also up for vote.
